Libraries/LibWeb/Painting/PaintableBox.cpp

```diff
@@ -1049,15 +1049,17 @@ Paintable::DispatchEventOfSameName PaintableBox::handle_mousemove(Badge<EventHan
 
 void PaintableBox::handle_mouseleave(Badge<EventHandler>)
 {
-    auto previous_draw_enlarged_horizontal_scrollbar = m_draw_enlarged_horizontal_scrollbar;
-    m_draw_enlarged_horizontal_scrollbar = false;
-    if (previous_draw_enlarged_horizontal_scrollbar != m_draw_enlarged_horizontal_scrollbar)
-        set_needs_display();
-
-    auto previous_draw_enlarged_vertical_scrollbar = m_draw_enlarged_vertical_scrollbar;
-    m_draw_enlarged_vertical_scrollbar = false;
-    if (previous_draw_enlarged_vertical_scrollbar != m_draw_enlarged_vertical_scrollbar)
-        set_needs_display();
+    if (!m_scroll_thumb_dragging_direction.has_value()) {
```

Comment:
Instead of indenting the entire function under an if statement, let's use an early return:
```c++
if (m_scroll_thumb_grab_position.has_value())
    return;
```

Reply:
No problem. I just pushed the change.  See if I did it correctly.